CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Due to [\#5553](https://github.com/decidim/decidim/pull/5553), SSL is turned on
 
 ### Added
 
+- **decidim-participatory_processes**: Add related assemblies to participatory processes [\#5868](https://github.com/decidim/decidim/pull/5868)
+- **decidim-core**: Allow users to register with a preferred language. [\#5789](https://github.com/decidim/decidim/pull/5789
```

Comment:
All seems ok now @armandfardeau , thanks for your efford!! :smile: 
If you could resolve changelog file, all work would be done! 
Thanks!!